### PR TITLE
Add disconnect

### PR DESCRIPTION
**Pull request by smyzin**
- updated dropdown
- added disconnect button
- added wallet options

### CODE REVIEW

File: src/components/Dropdown/Menu.tsx [hunk 1]
```diff
@@ -16,15 +17,20 @@ import classnames, {
...
   classnames(
     position('absolute'),
     inset(
-      'top-7',
+      { 'top-7': !extraSpacing },
       fitToItemSize
         ? { 'right-0': true, 'xs:right-auto': true, 'xs:left-0': true }
         : '-left-2.5'
     ),
+    extraSpacing ? margin('mt-2.5') : undefined,
```
Comment (T-Damer) [inline src/components/Dropdown/Menu.tsx:33]: Let's do an object here as well

File: src/components/navbar/RightBlock.tsx [hunk 2]
```diff
@@ -52,7 +52,22 @@ const AccountContainer = ({
   needNetworkChange: boolean
   eNSName?: string
 }) => {
+  const [, setLocation] = useLocation()
   const { isBurner } = useBadgeAccount()
+  const onSelectOption = (option: string) => {
+    switch (option) {
```
Comment (T-Damer) [inline src/components/navbar/RightBlock.tsx:58]: Hmm, maybe we can use `enum` here 🤔

File: src/components/navbar/RightBlock.tsx [hunk 3]
```diff
@@ -52,7 +52,22 @@ const AccountContainer = ({
...
+      case 'disconnect':
+        BurnerWalletStore.burn()
+        setLocation('/')
+        break
+      case '/wallet':
```
Comment (T-Damer) [inline src/components/navbar/RightBlock.tsx:63]: Maybe we shouldn't use `/` in the cases?

**Review by T-Damer — COMMENTED** [review]
cool, but if we put everything into `walletOptions` and use it like `walletOptions[option]`, this will be very cool

**Review by T-Damer — APPROVED** [review]

File: src/components/navbar/RightBlock.tsx [hunk 4]
```diff
@@ -52,7 +52,22 @@ const AccountContainer = ({
   needNetworkChange: boolean
   eNSName?: string
 }) => {
+  const [, setLocation] = useLocation()
   const { isBurner } = useBadgeAccount()
+  const onSelectOption = (option: string) => {
```
[the remark this answers is not in the document]
Comment (smyzin) [reply src/components/navbar/RightBlock.tsx:57]: It's not an option (the wrong name). It should be called `selectedValue` instead. I've already fixed it

**Review by T-Damer — APPROVED** [review]

**Pull request merged**